FindLibJpegTurbo.cmake

```diff
@@ -18,14 +19,52 @@ find_path(LibJpegTurbo_INCLUDE_DIR
   PATHS ${LibJpegTurbo_PKGCONF_INCLUDE_DIRS} /opt/libjpeg-turbo/include $ENV{LIBJPEGTURBO_ROOT}/include
 )
 
+# Search for header with version: jconfig.h
+if(LibJpegTurbo_INCLUDE_DIR)
+  if(EXISTS "${LibJpegTurbo_INCLUDE_DIR}/jconfig.h")
+    set(_version_header "${LibJpegTurbo_INCLUDE_DIR}/jconfig.h")
+  elseif(EXISTS "${LibJpegTurbo_INCLUDE_DIR}/x86_64-linux-gnu/jconfig.h")
+    set(_version_header "${LibJpegTurbo_INCLUDE_DIR}/x86_64-linux-gnu/jconfig.h")
+  else()
+    set(_version_header)
+    if(NOT LibJpegTurbo_FIND_QUIETLY)
+      message(STATUS "Could not find 'jconfig.h' to check version")
+    endif()
+  endif()
+endif()
+
+# Found the header, read version
+if(_version_header)
+  file(READ "${_version_header}" _header)
+  if(_header)
+   string(REGEX REPLACE ".*#define[\t ]+LIBJPEG_TURBO_VERSION[\t ]+([0-9.]+).*"
+     "\\1" LibJpegTurbo_VERSION "${_header}")
+  endif()
+  unset(_header)
+endif()
```

Comment:
I'm not really sure. If the header can't be found, it means jconfig.h is installed in an unexpected location or missing altogether. Those are not good signs and the script will fail if it can't determine the version. Normally users should be able to rely on the LibJpegTurbo_VERSION output. The only benefit of setting a default value would be for users who have a broken or unusual install AND don't care about version checking, but then the only possible default value would be 0 or empty IMO.